agent/Makefile

```diff
@@ -20,16 +20,16 @@ GOSEC=v2.12.0
 GOLANGCI=v1.46.2
 GOSWAG=v1.8.3
 
-.PHONY: gow-crane
-gow-crane: ##
+.PHONY: go-crane
+go-crane: ##
 	cd cmd/crane && \
-	gow run . ; \
```

Comment:
The change is OK, I've suggested it, because it is not working, offers no benefit over basic go command as it is now. This could even be a first issue thing.